lib/services/twitter.rb

```diff
@@ -42,9 +42,9 @@ def receive_push
         url = commit['url']
         # Strip out leading @s so that github @ mentions don't become twitter @ mentions
         # since there's zero reason to believe IDs on one side match IDs on the other
-        message = commit['message'].split(' ').map do |word|
-          (word.length > 1 && word[0] == '@') ? "@\u200b#{word[1..word.length]}" : word
-        end.join(' ')
+        message = commit['message'].gsub(/\B@[^ ]+/) do |word|
```

Comment:
Total nit, but would `[[:graph:]]` be more accurate here? For example:

``` ruby
"foob @\tbar".gsub(/\B@[^ ]+/) { |word| puts word; "@X#{word[1..word.length]}" }
@	bar
=> "foob @X\tbar"
```

vs

``` ruby
"foob @\tbar".gsub(/\B@[[:graph:]]+/) { |word| puts word; "@X#{word[1..word.length]}" }
=> "foob @\tbar"
```